src/bot.coffee

```diff
@@ -162,14 +155,17 @@ class SlackBot extends Adapter
   # @private
   ###
   eventHandler: (event) =>
-    {text, rawText, user, channel, subtype, topic, bot, item} = event
+    {user, channel, bot} = event
 
     # Ignore anything we sent
     # NOTE: coupled to getting `rtm.start` data
-    return if (user && (user.id is @self.id)) || (bot && (bot.id is @self.bot_id))
+    return if (user && (user.id is @self.id || user.id is @self.bot_id)) || (bot && (bot.id is @self.bot_id))
```

Comment:
this is interesting, can `user.id === @self.bot_id`? when would that happen?

Reply:
With reaction events the user.id is the bot id